packages/Adapters/POD_C/src/DTK_C_API.h

```diff
@@ -0,0 +1,49 @@
+#ifndef DTK_NONAME_H
+#define DTK_NONAME_H
+
+#include "mpi.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// QUESTION: Do we want to be able to track missed entities?
+
+// Convenience typedef for an opaque pointer
+typedef void DTK_Map;
+
+// Possible data layouts:
+// * Contiguous 
+//     X1 X2 ... Xn Y1 Y2 ... Yn
+// * Interleave 
+//     X1 Y1 X2 Y2 ... Xn Yn
+typedef enum data_layout { DTK_CONTIGUOUS, DTK_INTERLEAVE } DTK_Data_layout;
```

Comment:
Instead of DTK_CONTIGUOUS let's use DTK_BLOCKED. The memory is always contiguous here and blocked represents that the data is blocked by dimension instead of interleaved by dimension.